src-electron/db/db-mapping.js

```diff
@@ -215,7 +215,8 @@ exports.map = {
       hasSpecificResponse: dbApi.toDbBool(x.RESPONSE_REF),
       isIncoming: x.INCOMING,
       isOutgoing: x.OUTGOING,
-      isDefaultResponseEnabled: x.IS_DEFAULT_RESPONSE_ENABLED
+      isDefaultResponseEnabled: x.IS_DEFAULT_RESPONSE_ENABLED,
+      isLargeMessage: dbApi.fromDbBool(x.IS_LARGE_MESSAGE)
```

Comment:
Is this the only one that the quality takes are bringing in for now in terms of commands?

Reply:
seems to be for now based of our discussion